src/Components/Web.JS/src/Platform/Circuits/CircuitManager.ts

```diff
@@ -162,13 +162,18 @@ function getComponentComment(commentNodeIterator: ComponentCommentIterator): Com
   }
 }
 
-function createComponentComment(json: string, start: Node, iterator: ComponentCommentIterator): ComponentComment {
+function createServerComponentComment(json: string, start: Node, iterator: ComponentCommentIterator): ComponentComment | undefined {
   const payload = JSON.parse(json) as ComponentComment;
   const { type, sequence, descriptor, prerenderId } = payload;
-  if (type !== 'server') {
+  if (type !== 'server' && type !== 'client') {
```

Comment:
Why would CircuitManager encounter this? Could you add a comment for this?

Reply:
You could have a page that renders both a server component and a client component, I guess we don't support that, but it's better to be defensive I think.